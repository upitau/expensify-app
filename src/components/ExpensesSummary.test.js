import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from './ExpensesSummary';

test('should correctly render ExpnseSummary with one expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={45678} />);
  expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpnseSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={12345678} />);
  expect(wrapper).toMatchSnapshot();
});