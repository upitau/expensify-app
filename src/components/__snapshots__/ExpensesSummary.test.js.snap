// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpnseSummary with multiple expenses 1`] = `
<div>
  <p>
    Viewing 
    23
     
    expenses
     totalling 
    $123,456.78
  </p>
</div>
`;

exports[`should correctly render ExpnseSummary with one expense 1`] = `
<div>
  <p>
    Viewing 
    1
     
    expense
     totalling 
    $456.78
  </p>
</div>
`;
